### PR TITLE
Listview: fix for many colored listviews on the same page

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

**Comment by TomaszLukawskiSam** [comment]
All tests pass on my local PC

File: src/js/profile/mobile/widget/mobile/Listview.js [hunk 1]
```diff
@@ -153,7 +153,11 @@
 						 * @property {boolean} options.coloredBackground=true enables/disables colored background
 						 */
 						options = {
-							coloredBackground: true
+							coloredBackground: true,
+							colorRestOfScreen: true,
+							firstColorStep: 0,
+							lastColorStep: 0,
+							disableColorChange: false
```
Comment (TomaszLukawskiSam) [inline src/js/profile/mobile/widget/mobile/Listview.js]: it will be removed

File: src/js/profile/mobile/widget/mobile/Listview.js [hunk 2]
```diff
@@ -188,12 +192,15 @@
 					// starting default color for gradient background
 					self._colorBase = [250, 250, 250, 1];
 					// color modifier for each background gradient step
-					self._colorStep = [0, 0, 0, -0.04];
+					self._colorStep = [0, 0, 0, -0.04] ;
```
Comment (TomaszLukawskiSam) [inline src/js/profile/mobile/widget/mobile/Listview.js]: to fix

**Comment by lslachciak** [comment]
I verified PR works well, but new issue not related to this PR was revealed

File: src/js/profile/mobile/widget/mobile/Listview.js [hunk 3]
```diff
@@ -352,23 +376,37 @@
...
 				return newElement;
 			};
 
+			prototype._getCanvas = function () {
+				var self = this,
+					canvas;
+
+				// If reference to canvas has been changed.
+				// Developer can remove canvas from list, eg. by element.innerHTML = "<li>item</li>"
+				if (self._context) {
```
Comment (lslachciak) [inline src/js/profile/mobile/widget/mobile/Listview.js]: shouldn't be 
(if self._context && !self.context.canvas) ?

Comment (TomaszLukawskiSam) [reply src/js/profile/mobile/widget/mobile/Listview.js]: You probably mean:
`if self._context && self._context.canvas`

This additional condition has no impact on current code.

**Pull request merged**